golem_messages/message/tasks.py

```diff
@@ -197,6 +197,14 @@ def assert_role(role, expected, actual):
         self.validate_ownership_chain(concent_public_key=concent_public_key)
         return True
 
+    @classmethod
+    def deserialize_with_header(cls, header, data, *args, **kwargs):  # noqa pylint: disable=arguments-differ
+        instance: SubtaskResultsAccepted = super().deserialize_with_header(
```

Comment:
@dybi the type is _not_ guaranteed to be `SubtaskResultsAccepted` here ...

Reply:
good catch, I simply moved it from `SubtaskResultsAccepted` without changing the type... my bad ;)